sources/Inputs/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2020, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+
+set(SOURCES
+  src/NetworkManager.cpp
+  src/Node.cpp
+)
+
+add_library(inputs STATIC ${SOURCES})
+
+target_include_directories(inputs
+PUBLIC
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+)
+target_link_libraries(inputs
+PUBLIC
```

Comment:
I don't know in this context but generally PRIVATE should be used by default unless there is a good reason for PUBLIC, reasons are listed here https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#transitive-usage-requirements

Reply:
It is public because I want to link the transitive libraries at level above

Reply:
Public is ok but because you include #include <DYNDataInterface.h> in a .h and then you need transitivity, if it would only be included in .cpp it should then be private